sklearn/cluster/_kmeans.py

```diff
@@ -1756,6 +1756,13 @@ class MiniBatchKMeans(_BaseKMeans):
         a value may cause convergence issues, especially with a small batch
         size.
 
+        When there are too few points in the dataset, some centers may be
+        duplicated, which means that a proper clustering in terms of the number
+        of requesting clusters and the number of returned clusters will not
+        always match. To solve this set `reassignment_ratio=0`, so that
+        reassignments to avoid too small of clusters when there are too few
+        data points is possible.
```

Comment:
```suggestion
        always match. One solution is to set `reassignment_ratio=0`, which
        prevents reassignments of clusters that are too small.
```

Also I think this should go into the "Notes" section a few sections down as a new paragraph:

https://github.com/scikit-learn/scikit-learn/blob/3a9e708782f3b9f650f7e9ba9c42051e3c472699/sklearn/cluster/_kmeans.py#L1801-L1803

If someone runs into this issue, I suspect they do not know about the `reassignment_ratio` parameter and would likely not look through every parameters' description for a solution.